frontend/src/common/helpers/disableMouseWheelInputOnNumberField.ts

```diff
@@ -0,0 +1,12 @@
+// Prevent mouse wheel from changing the value on fields where inputType: "number"
+export const disableMouseWheelInputOnNumberField = (
+  event: React.FocusEvent<HTMLInputElement>,
+) => {
+  event.target.addEventListener(
```

Comment:
Can this same effect be achieved in another way? Because as is, this method creates a side effect which mutates the existing event, and also we might need to remove the event listener later when the page/component lifecycle ends? I'm wondering if there's another way to override the mouse wheel behaviour (maybe by passing in a prop somewhere if that's possible) without introducing side effects...

Reply:
The simplest way would be to stop using inputType: "number" in favor of a standard input, like we are using for unitNumber for example. However if we are set on using the number field then this method of attaching the event listener to the DOM element is the only way I can solve this problem, since React doesn't offer direct control over this.

My original attempt to solve this issue was something like this:

`
export const disableMouseWheelInput = (
  event: React.WheelEvent<HTMLInputElement>
) => {
  event.preventDefault(); 
};

<CustomFormComponent onWheel={disableMouseWheelInput} />
`

However, this will not work as indicated by the following error in the browser console:

"Unable to preventDefault inside passive event listener invocation"

I do agree however that we should remove the event listener, which I have added in my latest commit